<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>repeaterSetup</class>
 <widget class="QMainWindow" name="repeaterSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Repeater Setup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="rptReadRigBtn">
        <property name="text">
         <string>Read Current Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="debugBtn">
        <property name="text">
         <string>Debug</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Repeater Duplex</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="rptSimplexBtn">
           <property name="text">
            <string>Simplex</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptDuplexBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rptDupPlusBtn">
           <property name="text">
            <string>Dup+</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptDuplexBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rptDupMinusBtn">
           <property name="text">
            <string>Dup-</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptDuplexBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rptAutoBtn">
           <property name="text">
            <string>Auto</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptDuplexBtns</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Repeater Tone Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="toneNone">
           <property name="text">
            <string>None</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptToneBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toneTone">
           <property name="text">
            <string>Transmit Tone only</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptToneBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toneTSQL">
           <property name="text">
            <string>Tone Squelch</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptToneBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toneDTCS">
           <property name="text">
            <string>DTCS</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptToneBtns</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Tone Selection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Tone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rptToneCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>DTCS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rptDTCSCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="rptDTCSInvertTx">
             <property name="text">
              <string>Invert Tx</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="rptDTCSInvertRx">
             <property name="text">
              <string>Invert Rx</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rptDuplexBtns"/>
  <buttongroup name="rptToneBtns"/>
 </buttongroups>
</ui>
